portfolio/src/main/java/com/google/sps/data/ScheduledTask.java

```diff
@@ -37,4 +39,8 @@ public boolean equals(Object other) {
   public static boolean equals(ScheduledTask a, ScheduledTask b) {
     return a.task.equals(b.task) && a.startTime.equals(b.startTime);
   }
+
+  public void setCompleteness(boolean scheduleStatus) {
```

Comment:
Consider renaming scheduledStatus to isCompletelyScheduled.

Then in the method content you can do,

this.isCompl... = isCompl...

Reply:
Updated.